theme/_layouts/default.html

```diff
@@ -9,17 +8,17 @@
 
     {% include nav.html %}
 
-    <main id="main" class="container" role="main">
+    <main id="main">
       {% unless page.layout == "docs" %}
-       <div class="container mt-5">
-         <h1 class="display-5 text-center">{{ page.title }}</h1>
+       <!-- <div class="container mt-5"> -->
+         <h1 class="wrapper">{{ page.title }}</h1>
```

Comment:
Why is there a `.wrapper` class on the `h1` element element?

Reply:
It was a lazy way to center the text on the page...